fedimint-server/src/consensus/mod.rs

```diff
@@ -766,6 +766,52 @@ impl FedimintConsensus {
         Ok(())
     }
 
+    async fn accepted_transaction_status(
+        &self,
+        txid: TransactionId,
+        accepted: AcceptedTransaction,
+        dbtx: &mut DatabaseTransaction<'_>,
+    ) -> TransactionStatus {
+        let mut outputs = Vec::new();
+        for (out_idx, output) in accepted.transaction.outputs.iter().enumerate() {
+            let outpoint = OutPoint {
+                txid,
+                out_idx: out_idx as u64,
+            };
+            let outcome = self
+                .modules
+                .get_expect(output.module_instance_id())
+                .output_status(
+                    &mut dbtx.with_module_prefix(output.module_instance_id()),
+                    outpoint,
+                    output.module_instance_id(),
+                )
+                .await
+                .expect("the transaction was processed, so should be known");
+            outputs.push((&outcome).into())
+        }
+
+        TransactionStatus::Accepted {
+            epoch: accepted.epoch,
+            outputs,
+        }
+    }
+
+    pub async fn wait_transaction_status(
```

Comment:
same thing about `wait` and `await` or `wait_for`. 
feel free to ignore. 

maybe some native English speaker can tell me whether that's correct; in which case I'll shut up about it in the future :)